templates/ts/action/.webappignore

```diff
@@ -0,0 +1,24 @@
+.webappignore
```

Comment:
Can you help check with @Siglud and @hund030 if this is webappignore file or appserviceignore file? Also please check the content in this file is updated to v5 folder structure.